### DISCUSSION TITLE
清理频道消息

### DISCUSSION THREAD

**Discussion by skrlst** [chanify/chanify#149] [Q&A] [answered]
随着时间的变化，频道内的消息也会跟着增加

想知道消息这块是怎么存储的？

是否占用本地空间？

如果占用本地空间，那么该如何清理呢？

> 我只在频道内找到了删除操作，是长按某一条消息，随后会弹出一个菜单，可以看到删除按钮

如果只能这样操作，可否在未来增加一个批量删除的功能，或者清空频道消息？

![image](https://github.com/chanify/chanify/assets/70947084/b4ea27c8-45de-4134-9325-284d44cbc614)

**Comment by skrlst** [comment]
多选删除的功能找到了，在 https://github.com/chanify/chanify/discussions/49 有说明。

那清空频道的操作有吗

**Reply by wizjin** [reply]
你好，在频道的消息列表页面，长按右上角“...”按钮，在弹出的菜单中有清除消息的功能。

**Comment by wizjin** [comment, marked as answer]
> 随着时间的变化，频道内的消息也会跟着增加
> 
> 想知道消息这块是怎么存储的？

答：消息是在手机上使用文件数据库 SQLite 存储的

> 是否占用本地空间？

答：占用本地空间，但是如果只是文本消息，占用空间很小

> 如果占用本地空间，那么该如何清理呢？

答：在频道的消息列表页面，长按右上角“...” 有清空该频道所有消息的功能
